src/api/app/views/webui2/webui/patchinfo/show.html.haml

```diff
@@ -0,0 +1,57 @@
+- @pagetitle = @package
+.card.mb-3
+  = render partial: 'webui2/webui/package/tabs', locals: { project: @project, package: @package }
+  .card-body
+    .patchinfo.row
+      .col-md-8
+        = patchinfo_header(@patchinfo, @pkg_names)
+        = render partial: 'webui/webui/collapsible_text', locals: { text: @patchinfo.description,
+                                                                    threshold: { text_length: 1000, line_count: 10 },
+                                                                    type: 'description' }
+      .col-md-4
+        = render partial: 'side_elements', locals: { patchinfo: @patchinfo, packager: @packager }
+
+      .col-md-12
+        - if User.current.can_modify?(@package)
+          %ul.list-inline.mb-0
+            %li.list-inline-item
+              = link_to(edit_patchinfo_path(project: @project, package: @package)) do
+                %i.fas.fa-edit.text-secondary
+                Edit patchinfo
+            %li.list-inline-item
+              = link_to('#', class: 'nav-link', data: { toggle: 'modal', target: '#delete-patchinfo-modal' }) do
+                %i.fas.fa-times-circle.text-danger
+                Delete patchinfo
+          = render partial: 'delete_patchinfo_modal', locals: { project: @project, package: @package }
+
+.patchinfo.row
+  - if @patchinfo.message.present?
+    .col-md-12
+      .card.mb-3
+        %h5.card-header Message
+        .card-body
+          = simple_format @patchinfo.message
+
+  .col-md-6
+    .card.mb-3
+      %h5.card-header Fixed bugs
+      .card-body
+        - if @patchinfo.issues.present?
+          - @patchinfo.issues.each do |issue|
```

Comment:
Please add the issue "summary" here. It helps the Security Maintenance team to track, at one glance, which issues were fixed on that incident

Reply:
I assume you are talking about this:
![Screenshot_2019-04-17 Open Build Service(10)](https://user-images.githubusercontent.com/1212806/56302987-4334a880-613b-11e9-9b7d-58b611fd1493.png)

This info is in `issue[3]`

Reply:
yes great!